src/components/layertree/LayerTree.js

```diff
@@ -152,12 +152,8 @@ class LayerTree extends PureComponent {
             isChecked: !value,
             isExpanded: !value,
           },
-          child => {
-            if (child.id === item.id || child.type === 'checkbox') {
-              return true;
-            }
-            return false;
-          },
+          child => child.id === item.id,
+          child => child.type === 'checkbox',
```

Comment:
Why 2 functions now? one is enough no?

Reply:
It is aimed to apply 'isIgnoredFunc' when re-applying to children, but without the filter on checkbox.